AWS/manifest.json

```
@@ -29,5 +29,5 @@
     "name": "AWS",
     "uuid": "b4462429-6f0f-42b5-87b8-430111697d28",
     "slug": "aws",
-    "version": "1.27"
+    "version": "1.28"
```

I suggest increasing as minor update
```suggestion
    "version": "1.27.1"
```